placeholder_images.py

```diff
@@ -0,0 +1,65 @@
+from wmfdata.spark import get_session
+import argparse
+import requests
+import pandas as pd
+import numpy as np
+import findspark
+
+findspark.init('/usr/lib/spark2')
+
+spark = get_session(type='regular', app_name="ImageRec-DEV Training")
+
+
+class PlaceholderImages:
+    def __init__(self, snapshot):
+        self.snapshot = snapshot
+
+    @staticmethod
+    def get_placeholder_category():
```

Comment:
This is outside the scope of this PoC, but we should add some instrumentation to this callback.

We should also consider some failure scenarios.
What happens if the API has an hiccup and returns significantly less records than expected? That'd be a valid response with potentially borked data.
Would it be better to about the transaction and rollback to a "stable" static list of images?

Reply:
I totally agree. Should this be a v1 task?

Reply:
Yep, let's save this for v1.